File: data/imported/knowledge/cells/java/merged/snippets/snippet_076.java
@Test
    void CV_30_getCellsGetStringAcceptsA1Notation() {
        Workbook wb = new Workbook();
        Cells cells = wb.getWorksheets().get(0).getCells();
        cells.get("A1").putValue(1);
        assertEquals(1, cells.get("A1").getValue());
    }

File: data/imported/knowledge/cells/cpp/merged/snippets/snippet_076.cpp
TEST(WorkbookPortingTests, SaveAndLoadStreamRoundTrip)
{
    Workbook workbook;
    PopulateBasicWorkbook(workbook);
    std::vector<std::uint8_t> output;

    workbook.Save(output, SaveOptions());
    ASSERT_GT(output.size(), 100u);

    Workbook loaded(output);
    EXPECT_EQ("Data", loaded.GetWorksheets()[0].GetName());
    EXPECT_EQ("alpha", loaded.GetWorksheets()[0].GetCells()["A1"].GetStringValue());
    EXPECT_EQ(42, loaded.GetWorksheets()[0].GetCells()["B1"].GetValue().AsInteger());
}